### ISSUE TITLE
Docs: Clarification Raw Cell Options

### ISSUE THREAD

**Issue by par2** [spatialaudio/nbsphinx#8]
I've used IPython for a couple years, but hadn't realized the raw cells had other formats.  It took some time for me to find in it IPython 3.  I suggest clarifying after selecting a cell:

1. First select Raw NBConvert
2. Then next to Cell Toolbar: select Raw Cell Format
3. Choose one of the format options
... 

These options may differ for Jupyer (IPython 4+).  BTW, this is a great extension.  Finding it this week has been most fortunate.

**Comment by mgeier** [comment]
Thanks for the report!

I guess you are talking about http://nbsphinx.readthedocs.org/en/0.2.2/example.html#Raw-Cells, right?

Can you please make a suggestion for concrete changes in wording?
You can also make a PR for that, if you like.

Are you actually using raw cells in combination with `nbsphinx`?
I implemented this just for completeness' sake, I'd be interested if anyone is actually using this for something meaningful.

**Comment by par2** [comment]
Yes, that link is what I am referring to.  

I thought I gave a specific suggestion.  What else are looking you looking for exactly?  I'm happy to help.

Yes indeed, I am using the raw cells to inject rst code or sphinx messages into my docs, i.e. such as notes, important and see also.  I not sure how else to do it.  The following examples illustrate converting rst code in Jupyter Notebooks to html using `nbsphinx` 0.2.2:

Normal rst code
```
.. note::

	Example - Note
	
.. important::

	Example - Important
	
.. seealso::

	Example - See Also
```

Code in IPython/Juypyter Notebook

![2016-01-16 21_40_13-test](https://cloud.githubusercontent.com/assets/15694632/12375646/0c9c3660-bc9a-11e5-9425-36ce5e77b997.png)

HTML Result: post `make html`
![sphinx](https://cloud.githubusercontent.com/assets/15694632/12375648/20086cf0-bc9a-11e5-9a37-a4da74e8bcd2.png)

**Comment by mgeier** [comment]
I meant something like "change the sentence 'blah blah whatever' to 'blah blah something else'".

Anyway, I think it would be best if you suggest your changes with a pull request.
I just split `example.ipynb` into three parts, so you should make your suggested changes to the file https://github.com/spatialaudio/nbsphinx/blob/master/doc/raw-cells.ipynb.

It would be great if you could include your Jupyter screenshot there.

Thanks for describing you use case. I think it would be best if the Jupyter Notebook would allow elements like "Note" and "See Also" directly in Markdown, but AFAIK, that's (currently) not possible (except probably with extensions or local CSS manipulations).
So indeed raw reST cells are the solution here, but sadly, this makes the notebook files less nice to look at in Jupyter itself.

**Comment by pylang** [comment]
Pull request https://github.com/spatialaudio/nbsphinx/pull/13 submitted.

**Comment by mgeier** [comment]
Closed by #13.
